### PR TITLE
Add AssistedUILibVersion component

### PR DESCRIPTION
**Pull request by mareklibra**
The library version is stored in the DOM:

```
<div data-test-id="assisted-ui-lib-version" hidden="">1.5.2-1</div>
```

Can be used by Cypress CI for better reporting.

### CODE REVIEW

**Comment by mareklibra** [comment]
Cc @cdvultur

File: src/components/Router.tsx [hunk 1]
```diff
@@ -5,27 +5,31 @@ import { Clusters, ClusterPage, NewClusterPage } from './clusters';
...
-    <Switch>
-      <Route path={`${routeBasePath}/clusters/~new`} component={NewClusterPage} />
-      <Route path={`${routeBasePath}/clusters/:clusterId`} component={ClusterPage} />
-      <Route path={`${routeBasePath}/clusters`} component={Clusters} />
-      {children}
-      <Redirect to={`${routeBasePath}/clusters`} />
-    </Switch>
-  );
+export const Router: React.FC = ({ children }) => (
+  <AssistedUILibVersion>
```
Comment (jtomasek) [inline src/components/Router.tsx:17]: Is it necessary for the component to wrap the contents of a router? The component will need to be placed somewhere else in OCM once the cluster list/detail integration work is finished.

Comment (mareklibra) [reply src/components/Router.tsx:17]: Just a short-cut. I will move it somewhere down.

**Review by jtomasek — APPROVED** [review]

**Pull request merged**